.gitignore

```diff
@@ -1,3 +1,3 @@
 .vscode
 js-coverage
-node_modules
\ No newline at end of file
+node_modules
```

Comment:
Looks like there are some white space changes? Can you revert the changes?

Reply:
done